dipy/tracking/tests/test_streamline.py

```diff
@@ -161,6 +161,8 @@
 )
 
 streamline_64bit = streamline.astype(np.float64)
+streamline_16bit = streamline.astype(np.float16)
+streamline_16bit = streamline.astype(np.float16)
```

Comment:
```suggestion
```

Reply:
Thanks!

Reply:
The duplicate is back.

Reply:
sorry for that, during my rebase, I forgot to git pull before....